### ISSUE TITLE
Option to save exported script to file 

### ISSUE THREAD

**Issue by dfalster** [richfitz/remake#27]
Is it possible to add a feature to 

```
m <- maker::maker()
m$script()
```

so that can save script directly to file, similar to writeLines

```
writeLines <- function (text, con = stdout(), ...) 
{
    if (is.character(con)) {
        con <- file(con, "w")
        on.exit(close(con))
    }
 ...
}
```

**Comment by richfitz** [comment]
Good idea.  Need to think of a name is all (I don't think there is a generic we can easily overload).

**Comment by richfitz** [comment]
...output filename to `make_script` is most obvious choice I guess...

**Comment by dfalster** [comment]
`remake.R`

**Comment by richfitz** [comment]
Good idea :)

I was wondering about reserving that for R-based remake construction, but it could work for sure.  I'll add a filename option to `make_script` at least for now though.

**Comment by richfitz** [comment]
Fixed in 91e3e72f6a29936a1bd0c075f60018d169578b99
